src/sections.js

```diff
@@ -40,6 +40,31 @@ const examples = (section, context) => {
     const delimiter = arguments.getDelimiterValue(
         section, context.regexp.delimiter);
     context.delimiter = delimiter ? delimiter : context.delimiter;
+    parseArgs(section, context);
+};
+
+/**
+ * Parse argument property isFlag by examples section
+ * @param {string} section - examples section
+ * @param {object} context - internal config
+ */
+const parseArgs = (section, context) => {
+    section.split('\n').forEach((string) => {
+        const matches = string.match(context.regexp.examplesArgument);
```

Comment:
We can use global regex with multiline

Reply:
I suggest to remove this function to `arguments.js` file

Reply:
Please use pure function and think out it for this implementation. You can, i know it :)